src/styles/reset.scss

```diff
@@ -0,0 +1,13 @@
+
+body {
+  margin: 0;
+}
+
+* {
```

Comment:
this rules with `*` should be on the first line

Reply:
agree